docs/index.adoc

```diff
@@ -4798,26 +4799,27 @@ edit [--open[=<editor>]] <file>
 One of the <<Optional Parameter Limitations, limitations of options with an optional parameter>> is that they are difficult to combine with positional parameters.
 
 With a custom parser plugin, we can customize the parser, such that `VALUE` in `--option=VALUE` is interpreted as the option parameter, and in `--option VALUE` (without the `=` separator), VALUE is interpreted as the positional parameter.
-The code below demonstrates:
+The code below demonstrates this:
 
 .Java
 [source,java,role="primary"]
 ----
 @Command(name = "edit")
 class Edit {
 
-    @Parameters(index = "0", description = "The file to edit.")
+    @Parameters(index = "0", arity="0..1", description = "The file to edit.")
     File file;
 
-   enum Editor { defaultEditor, eclipse, idea, netbeans }
+   enum Editor { eclipse, idea, netbeans }
```

Comment:
I think we should keep `defaultEditor`: it is good to have a separate value to represent the absence of choice by the end user; I am imagining that in the application this would come from some configuration (so not hard-coded).